### ISSUE TITLE
cant enable the plugin

### ISSUE THREAD

**Issue by wheatzman** [ramokz/phantom-camera#194]
### Issue description

I can't enable the plugin on Godot 4.2.1 on macos. I get this errors (plugin version 0.6.2.2):

`  Unable to open file: res://.godot/imported/PhantomCameraIcon2D.svg-ff639bc802b50f88263b8dee8a265dab.ctex.
  Failed loading resource: res://.godot/imported/PhantomCameraIcon2D.svg-ff639bc802b50f88263b8dee8a265dab.ctex. Make sure resources have been imported by opening the project in the editor at least once.
  Failed loading resource: res://addons/phantom_camera/icons/PhantomCameraIcon2D.svg. Make sure resources have been imported by opening the project in the editor at least once.
  Unable to open file: res://.godot/imported/PhantomCameraIcon3D.svg-2bc62e479ff60e942df3edb50acf2cff.ctex.
  Failed loading resource: res://.godot/imported/PhantomCameraIcon3D.svg-2bc62e479ff60e942df3edb50acf2cff.ctex. Make sure resources have been imported by opening the project in the editor at least once.
  Failed loading resource: res://addons/phantom_camera/icons/PhantomCameraIcon3D.svg. Make sure resources have been imported by opening the project in the editor at least once.
  Unable to open file: res://.godot/imported/PhantomCameraHostIcon.svg-97309b98913e7760fc2cc984f77a836d.ctex.
  Failed loading resource: res://.godot/imported/PhantomCameraHostIcon.svg-97309b98913e7760fc2cc984f77a836d.ctex. Make sure resources have been imported by opening the project in the editor at least once.
  Failed loading resource: res://addons/phantom_camera/icons/PhantomCameraHostIcon.svg. Make sure resources have been imported by opening the project in the editor at least once.
  res://addons/phantom_camera/plugin.gd:31 - Parse Error: Could not preload resource file "res://addons/phantom_camera/icons/PhantomCameraIcon2D.svg".
  res://addons/phantom_camera/plugin.gd:32 - Parse Error: Could not preload resource file "res://addons/phantom_camera/icons/PhantomCameraIcon3D.svg".
  res://addons/phantom_camera/plugin.gd:33 - Parse Error: Could not preload resource file "res://addons/phantom_camera/icons/PhantomCameraHostIcon.svg".
  modules/gdscript/gdscript.cpp:2788 - Failed to load script "res://addons/phantom_camera/plugin.gd" with error "Parse error". (User)`

### Steps to reproduce

use Macos Sonoma 14.2.1, Godot version 4.2.1 and the plugin version 0.6.2.2

-> install the plugin like recommended on the documentation. Doesn't matter if via library or github.
-> when you want to enable the plugin in the project setting you get the error, also the console shows this errors above. 
-> it is not possible to enable the plugin

**Comment by ramokz** [comment]
Strange, just tried creating two new Godot (v. 4.2.1) projects on a Mac, and installed the addon via the `assetlib` on one, and via a `GitHub` release on the other. Didn't see any issues when enabling the addon or when running an example scene in either project.

It does sound familiar to #177, and a problem I've seen previously.

Does your Mac have an M-series chips, or one of the older Intel ones?

**Comment by wheatzman** [comment]
Oh I tested it now with a new project with Forward + options on and there it works fine. Could it be that in compatible mode for graphics the plugin is not working?

**Comment by wheatzman** [comment]
Ok yes its related to the render mode, tested it, and in case of compatible mode i am not able to enable the plugin and than this errors appear

**Comment by ramokz** [comment]
I suspect it's a Mac issue with Godot, though I'm not sure what the exact cause is.

> Could it be that in compatible mode for graphics the plugin is not working?

Yes, it seems to be exclusive to `Compatability mode` on Mac. Have not been able to replicate it on Linux, at least.

From a quick test, it seems like installation is only a problem if done so when in Compatibility mode. If it is installed while in `Forward+`, then switch to `Compatability` it looks like it's running fine. That said, the 3D example scenes don't load, as mentioned in #177.

I did also experience an issue with importing `.svg` files a while back, where I had to reimport them manually to resolve it. But hasn't appeared since then.

This fundamentally sounds like an engine bug, but it's not ideal still, given it's an issue that happens so early on in the process. Am not entirely sure if there's a way around that aside from waiting for a Godot fix, unfortunately...
